advertools/spider.py

```diff
@@ -642,7 +643,7 @@ class SEOSitemapSpider(Spider):
     }
 
 
-    def __init__(self, url_list, follow_links=False,
+    def __init__(self, url_list, meta, follow_links=False,
```

Comment:
This should be an optional parameter, so `meta=None`. 
I think it's better to keep it at the end of the signature, because it's not one of the essential parameters.

Reply:
Agreed.